### ISSUE TITLE
could you add support of the new optimizer: Muon

### ISSUE THREAD

**Issue by lrisguan** [tensorflow/tensorflow#97187]
### Issue type

Support

### Have you reproduced the bug with TensorFlow Nightly?

Yes

### Source

source

### TensorFlow version

2.19

### Custom code

No

### Current behavior?

In tensorflow 2.19, without the support of the new optimizer of Muon
Here's the link: [Muon](https://github.com/KellerJordan/Muon)

### Standalone code to reproduce the issue

```shell
No support of optimizer: Muon
```

### Relevant log output

```shell

```

**Comment by mihaimaruseac** [comment]
Can you send a PR please? We can review it.

**Comment by lrisguan** [comment]
I've sent one, but as I'm a green hand, maybe it needs your maintainers' help to fix it.
here's the link: [PR](https://github.com/tensorflow/tensorflow/pull/97231)

**Comment by Venkat6871** [comment]
Hi @lrisguan,

Apologies for the delay, and thanks for your patience.

Muon support is now available through Keras 3 and works with the TensorFlow backend. The original request was made against TensorFlow 2.19, before Muon was available in Keras. Muon was added to Keras in [[PR #21037](https://github.com/keras-team/keras/pull/21037)], and TensorFlow backend compatibility was fixed in [[PR #21924](https://github.com/keras-team/keras/pull/21924)].

With current TensorFlow/Keras releases, you can use:

```python
from tensorflow import keras

optimizer = keras.optimizers.Muon(learning_rate=0.01)
```

This has been verified with TensorFlow 2.21.0 / Keras 3.15.1, where `tf.keras.optimizers.Muon` successfully applies gradient updates.

Please upgrade to the latest stable TensorFlow/Keras versions and use the Keras 3 implementation. The older [[TensorFlow PR #97231](https://github.com/tensorflow/tensorflow/pull/97231)] is no longer needed because optimizer development is handled in Keras 3.

Please try this with the latest versions and let us know whether you are still seeing any issues.

Thank you!

**Comment by lrisguan** [comment]
LGTM! Thanks, no issues.
